# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/serviceworkers/FetchEventOpProxyChild.h
# Commit: cf972fe0d961
# Full Hash: cf972fe0d96164788f998f04008930474526e400
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2022-02-26 21:36:36
# Regressor Bug: 1754365
# File Overlap Count: 1
# Description:
#   Bug 1754365 - Create IPC for transferring NavigationPreload's PerformanceTimingData. r=dom-worker-reviewers,jesup
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D138251
# ==============================================================================

diff -r f8afd25bf41e -r cf972fe0d961 dom/serviceworkers/FetchEventOpProxyChild.h
--- a/dom/serviceworkers/FetchEventOpProxyChild.h	Sat Feb 26 08:40:25 2022 +0000
+++ b/dom/serviceworkers/FetchEventOpProxyChild.h	Sat Feb 26 08:40:26 2022 +0000
@@ -42,7 +42,7 @@
   ~FetchEventOpProxyChild() = default;
 
   mozilla::ipc::IPCResult RecvPreloadResponse(
-      ParentToChildInternalResponse&& aResponse);
+      ParentToChildResponseWithTiming&& aResponse);
 
   void ActorDestroy(ActorDestroyReason) override;
 